core/server/models/webhook.js

```diff
@@ -7,6 +7,10 @@ let Webhook,
 Webhook = ghostBookshelf.Model.extend({
     tableName: 'webhooks',
```

Comment:
We put default values to the Bookshelf model too.
See [explanation](https://github.com/TryGhost/Ghost/blob/2.2.3/core/server/models/post.js).